armi/physics/neutronics/globalFlux/globalFluxInterface.py

```diff
@@ -229,9 +226,10 @@ def interactCoupled(self, iteration):
         GlobalFluxInterface.interactCoupled(self, iteration)
 
     def getTightCouplingValue(self):
-        if self.tightCouplingConvergeOn == "keff":
+        """Return the parameter value"""
+        if self.TIGHT_COUPLING_CONVERGENCE_VARIABLE == "keff":
             return self.r.core.p.keff
-        if self.tightCouplingConvergeOn == "power":
+        if self.TIGHT_COUPLING_CONVERGENCE_VARIABLE == "power":
             scaledCorePowerDistribution = []
             for a in self.r.core.getChildren():
                 scaledPower = []
```

Comment:
Oops. This should be `self.coupler.parameter` now.

Reply:
https://github.com/terrapower/armi/pull/1033/commits/beceef774632726cfb3941590fef61a2d5d8183d